spec/index.html

```diff
@@ -4859,16 +4851,16 @@ <h4>Effective Boolean Value (EBV)</h4>
       </section>
       <section id="OperatorMapping">
         <h3>Operator Mapping</h3>
-        <p>The SPARQL grammar identifies a set of operators (for instance, <span class=
-                                                                                 "token">&amp;&</span>, <span class="token">*</span>, <span class="token">isIRI</span>) used
+        <p>The SPARQL grammar identifies a set of operators 
+          (for instance, <span class="token">&amp;&</span>, <span class="token">*</span>, <span class="token">isIRI</span>) used
```

Comment:
```suggestion
          (for instance, <span class="token">`&amp;&amp;`</span>, <span class="token">`*`</span>, <span class="token">`isIRI`</span>) used
```

Reply:
Changed to using `<code><span>...</span></code>`.

This keeps the HTML pattern predictable in case we need to find en masse. Elsewhere in this PR, `<code>` has been moved outside `<span>`.

Unfortunately, it seems backticks don't work around backup and it has to be `<code>` which is a shame.

And also backtick is a discussed character in the text!

Reply:
I'm guessing you meant to say, "Changed to use ``` ` ``` consistently." which requires wrapping the single backtick (and surrounding spaces) with triple backticks, as shown in this line (which itself is preceded and followed by a line of three tildes) —
~~~
Changed to use ``` ` ``` consistently.
~~~
Markdown has many complications to enable its simplicities...